src/core/public/http/http_service.test.ts

```diff
@@ -505,6 +559,112 @@ describe('interception', () => {
       'Response 3',
     ]);
   });
+
+  it('should not have access to request via response error interceptor when request error', async () => {
+    expect.assertions(3);
+
+    http.intercept({
+      response() {
+        throw new Error('Internal Server Error');
+      },
+    });
+    http.intercept({
+      responseError({ request }) {
+        expect(request).toBeUndefined();
+      },
+    });
+
+    await expect(http.fetch('/my/path')).rejects.toThrow();
+    expect(fetchMock.called()).toBe(true);
+  });
+
+  it('should have access to request via response error interceptor when no request error', async () => {
+    expect.assertions(3);
+
+    http.intercept({
+      response() {
+        throw new Error('Internal Server Error');
+      },
+    });
+    http.intercept({
+      responseError({ request }) {
+        expect(request).toBeDefined();
```

Comment:
Other than this line, I don't see the difference between this test and the one above it? Why would `request` be undefined in the other case?